src/node/build/buildPluginHtml.ts

```diff
@@ -67,11 +69,35 @@ export const createBuildHtmlPlugin = async (
     }
   }
 
+  const injectSystemJs = (html: string) => {
+    let tag: string
+    if (useSystemJs === false) {
+      return html
+    } else if (useSystemJs === true) {
```

Comment:
Put this condition after the `typeof useSystemJs === 'string'` block 
then you can just do `if (useSystemJs)`